doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst

```diff
@@ -117,8 +117,10 @@ The upgradable bootloader carries out the same signature and metadata verificati
 Also, it can upgrade both itself and the following image in the boot sequence, which, in most cases, is an application.
 
 .. caution::
-   You should add a second-stage bootloader only when necessary by the design or firmware upgrade needs.
-   Adding the second stage bootloader for no reason will lead to a degradation of the system's overall security, as attackers can exploit bugs that may exist in either bootloader.
+   * You should add a second-stage bootloader only when necessary by the design or firmware upgrade needs.
+     Adding the second stage bootloader for no reason will lead to a degradation of the system's overall security, as attackers can exploit bugs that may exist in either bootloader.
+   * When adding a second-stage bootloader, make sure to define :ref:`static partition for the immutable bootloader <ug_pm_static>`.
```

Comment:
Static partitioning is not only needed for second-stage bootloaders, but also for single-stage bootloaders.